lib/csdl2openapi.js

```diff
@@ -1025,7 +1025,7 @@ module.exports.csdl2openapi = function (
         const properties = propertiesOfStructuredType(entityType);
 
         keys.forEach((key, index) => {
-            const suffix = level > 0 ? '-' + level : '';
+            const suffix = level > 0 ? '_' + level : '';

```

Comment:
OpenAPI spec does not state any restrictions on parameter names, and dashes in headers are common, so why not in path template parameters?

To really make this work we need to revert https://github.com/oasis-tcs/odata-openapi/pull/91

Reply:
Yes, the specification is kind of loose. Probably to add more flexibility. I think most tools actually will handle this correctly. But I'm seeing some that are not really handling the `-` correctly